### ISSUE TITLE
Chris P.: `Field64` seems too small for robustness

### ISSUE THREAD

**Issue by cjpatton** [junyechen1996/draft-chen-cfrg-vdaf-pine#5]
In our analysis of Prio3 (https://eprint.iacr.org/2023/130) we found that robustness requires a field large enough to overcome a computationally bounded attacker that can do offline precomutation with the PRG.

The key term is the multipliciative loss via RO queries (the first term): 
![image](https://github.com/junyechen1996/draft-chen-cfrg-vdaf-pine/assets/3453007/54ed04af-60d0-45fb-a11c-4daa8035b72d)

Basically, if the soundness error is $\epsilon = 2^{128}$, then allowing $2^{64}$ RO queries drops the robustness bound to at least $2^{64}$. We believe this bound is relatively tight.

Repeating the verification step multiple times might help, but I think you would also need independent joint randomness for each trial.

I would recommend going with a 128-bit field, i.e., `Field128`.
